[Feat]#2 scheme파일 오류 수정

Willowwryu opened the pull request:
-dev브랜치에서 새로 클론
- 버튼 ui및 더미 펑션 구현
- 카메라모델, 카메라 모델 파일 분리

## 관련 이슈들
- #2 

## 작업 내용
- 셔터, 전후면 변경, 플래시 버튼 구현

## 리뷰 노트
-

## 스크린샷(UX의 경우 gif)
![Simulator Screen Recording - iPhone 14 Pro - 2023-03-01 at 15 00 09](https://user-images.githubusercontent.com/103012086/222107393-4290a568-47b1-49fd-b14b-f97f849a293b.gif)


## Reference
- 참고한 사이트, 정리한 링크를 달아주세요.

## 체크리스트
- [ ] 올바른 브랜치로 PR을 날리고 있는지 더블CHECK!
- [ ] 확인을 위해 바꾼 SceneDelegate.swift를 원래의 상태로 바꾸어 놓으셨나요?
- [ ] 불필요한 주석과 프린트문은 삭제가 되었나요?

Willowwryu committed:
[Feat]#2 scheme파일 오류 수정

-dev브랜치에서 새로 클론
- 버튼 ui및 더미 펑션 구현
- 카메라모델, 카메라 모델 파일 분리

Willowwryu committed:
[feat]Infoplist 카메라 권한 수정

- 카메라 권한요청 메세지 수정

sunshiningsoo commented on IAteIt/IAteIt/View/Upload/UploadView.swift:
```diff
@@ -6,14 +6,110 @@
 
-struct UploadView_Previews: PreviewProvider {
+
+struct CameraPreviewView: UIViewRepresentable {
```
[P1] 이 부분도 파일로 빼놓는 편이 좋을 것 같아요!

Willowwryu replied:
헉 이부분은 카메라 뷰로 UI와 같이 있는게 좋겠다고 생각했는데 굳이 분리 해야할까요?

sunshiningsoo replied:
저는 CameraPreviewView가 실제로는 CameraViewModel의 init()에서만 사용되고 있어서 해당 파일과 큰 연관성은 없어보인다고 생각했슴다!

Willowwryu replied:
해당 부분이 카메라 화면이 나오게 하는 부분이라 그랬는데 따로 파일 빼놓겠습니다!

Willowwryu committed:
[Feat] UI변경점 반영

- 전후면 버튼 제거
- 셔터 버튼 제거

eunbkang commented on IAteIt/IAteIt/View/Upload/CameraViewModel.swift:
```diff
@@ -0,0 +1,40 @@
+        model.requestAndCheckPermissions()
+    }
+    
+    func switchFlash() {
+        isFlashOn.toggle()
+    }
```
지난 회의에 따라 이 부분도 필요없을 것 같습니다!
아래 changeCamera()도 마찬가지로요!

Willowwryu replied:
넵 혹시나 나중에 필요할까봐 남겨둘까 했는데 그냥 모두 삭제 하겠습니다..!

eunbkang reviewed (approved):
수고하셨습니다:)

사진 frame 미리 보여주는 UI 같은 경우는 피그마에 작업해보긴 했는데 수욜에 한번 얘기해보시죠!

Willowwryu committed:
Merge branch 'dev' into origin/feat/#2.1-CameraView

Willowwryu commented on IAteIt/IAteIt/View/Upload/Camera.swift:
```diff
@@ -0,0 +1,60 @@
+                }
+                session.startRunning() // 세션 시작
+            } catch {
+                print(error) // 에러 프린트
```
옙 안납니다. https://twih1203.medium.com/swift-예외처리-throws-do-catch-try-하기-c0f320e61f62 참고해보세요!

heokyeol replied:
잘 봤습니다 !! 좋은 자료 감사합니다

sunshiningsoo commented on IAteIt/IAteIt.xcodeproj/project.pbxproj:
```diff
@@ -433,11 +443,14 @@
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"IAteIt/Preview Content\"";
+				DEVELOPMENT_TEAM = NH6966C938;
 				ENABLE_PREVIEWS = YES;
```
Development Team None으로 해서 올려주세욥! 불편해도 우선은 None으로 해서 올려야 할 것 같아요!ㅠ

Willowwryu replied:
넵 확인했슴다

Willowwryu committed:
[Fix] 펑션 제거 및 파일 분리

- struct_CameraPreviewView 파일로 분리
- 프로젝트 파일 사이닝 none으로 변경
- 필요없어진 플래시, 전후면 변경 펑션 제거
